src/integrations/hapi.js

```diff
@@ -4,7 +4,7 @@ import { JwksClient } from '../JwksClient';
 const handleSigningKeyError = (err, cb) => {
   // If we didn't find a match, can't provide a key.
   if (err && err.name === 'SigningKeyNotFoundError') {
-    return cb(null, null, null);
+    return cb(err, null, null);
```

Comment:
Don't reckon you need the `if` condition because you're doing the same thing in both cases